setup.cfg

```diff
@@ -26,7 +26,8 @@ install_requires =
     psutil>=5.7.2
     numpy>=1.16
     astropy>=4.0
-    romanad==0.4.0
+    # romanad>=0.4.0
+    romanad @ git+https://github.com/PaulHuwe/rad.git@RAD-25_DQInit_Calstep
```

Comment:
this is what's breaking the tests. Wasn't the PR on RAD merged? If so, update to rad/master so that checks pass here

Reply:
Explicitly pointed to main now, and the tests pass.